superset-frontend/src/components/dataViewCommon/TableCollection.tsx

```diff
@@ -272,6 +276,7 @@ export default function TableCollection({
             const rowId = row.original.id;
             return (
               <tr
+                data-test="table-row"
```

Comment:
I'm a little less enthused about these table-row and table-cell ones, since they don't add any specificity to the test, but I guess they don't hurt, meaning we can remove/alter the class if needed. In this case, it just feels like a little bloat to the code. But... for anyone else reading this, the plan is to remove these using [this package](https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id).

Reply:
hmm, how would we automate testing production/staging releases? I don't think these are really adding much bloat in the grand scheme of things. Data test attributes should be unique though, so I'd suggest changing this to `table-row-${i}` where I is the row number, so it's easier to select the nth row

Reply:
I was thinking about this but finally decided to left this as table-row and logic related to choosing proper element move to cypress code using children() and eq() syntax. What do you think? which option is more valuable?

Reply:
🤷 this is probably fine for now. We can easily switch if needed

Reply:
Interesting point (and thanks for the comments in general), @nytai - I agree this is fine for the purposes of this PR... let's have a separate chat/PR where we talk about these next steps.